Rails/Ruby (Rails).sublime-syntax

```diff
@@ -6,6 +6,7 @@ scope: source.ruby.rails
 extends: Packages/Ruby/Ruby.sublime-syntax
 
 file_extensions:
+  - arb
```

Comment:
The first entry is what is provided as default file extension when saving files. Hence I'd probably keep `ruby.rails` as safe default rather than a special extension.

Reply:
I'm agree, I thought in alphabetical order.